src/components/autocomplete/OfficerNameAutocomplete/index.tsx

```diff
@@ -4,7 +4,7 @@ import { BaseAutocomplete } from '../base';
 
 import { getMultipleUsers, getUserNames } from '@Services/UserService';
 
-type OfficerNameData = {
+export type OfficerNameData = {
```

Comment:
should later standardize type vs interface for props - nbd though

Reply:
I'll probably be switching to using interface instead of type, since interface is more common in our codebase.